Dockerfile

```diff
@@ -1,14 +1,34 @@
-FROM docker.io/library/node:16.14.2 AS build
+FROM docker.io/library/node:16.14.2 AS web-builder
```

Comment:
Any reason not to use nodjs RH image or similar? https://catalog.redhat.com/software/containers/rhel8/nodejs-16-minimal/615aefc7c739c0a4123a87e2